config/initializers/recaptcha.rb

```diff
@@ -1,5 +0,0 @@
-Recaptcha.configure do |config|
-  config.site_key   = ENV['RECAPTCHA_SITE_KEY'] ||  '6LeIxAcTAAAAAJcZVRqyHh71UMIEGNQ_MXjiZKhI'
```

Comment:
Removing this file will break Heroku badly, it needs to stay in the repository. Otherwise, on every deploy Heroku will delete this file.